java-example/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'java'
+
+buildscript {
+    repositories {
+        flatDir { dirs 'lib' }
+    }
```

Comment:
What does this do? :thinking:

Reply:
Makes Gradle understand that it may find
`dependencies { classpath 'com.cossacklabs.themis:com.cossacklabs.themis-0.13.1' }`
in `lib` dir. Otherwise it complains about no repos so it couldn't find that classpath.